nhrpd/nhrp_nhs.c

```diff
@@ -96,7 +99,7 @@ static void nhrp_reg_reply(struct nhrp_reqid *reqid, void *arg)
 	c = nhrp_cache_get(ifp, &p->dst_proto, 1);
 	if (c)
 		nhrp_cache_update_binding(c, NHRP_CACHE_NHS, holdtime,
-					  nhrp_peer_ref(r->peer), 0, NULL);
+					  nhrp_peer_ref(r->peer), htons(mtu), NULL);
 }
```

Comment:
ntohs ?

Reply:
Yes this is a typo. Works because the implementation is typically the same for both on most architectures.

Reply:
Code has been changed to use ntohs